engines/tinsel/metaengine.cpp

```diff
@@ -76,6 +76,8 @@ class TinselMetaEngine : public AdvancedMetaEngine {
 	int getMaximumSaveSlot() const override;
 	SaveStateDescriptor querySaveMetaInfos(const char *target, int slot) const override;
 	void removeSaveState(const char *target, int slot) const override;
+
+	// TODO: Add getSavegameFile()
```

Comment:
Why add this comment? It seems to me that the default implementation of `getSavegameFile()` should work for this engine?

Reply:
See the comments in `loadGameState` and `removeSaveState`.